phase-1_combine-portfolios.R

```diff
@@ -159,15 +159,17 @@ saveRDS(data, file.path(output_dir, paste0(project_prefix, "_full.rds")))
 meta_output_dir <- file.path(output_dir, "meta")
 dir.create(meta_output_dir, showWarnings = FALSE)
 
-dir_create(file.path(meta_output_dir, pacta_directories))
+for (dir in file.path(meta_output_dir, pacta_directories)) {
+  dir.create(dir)
+}
 
 data %>%
   mutate(portfolio_name = "Meta Portfolio") %>%
   mutate(investor_name = "Meta Investor") %>%
   select(investor_name, portfolio_name, isin, market_value, currency) %>%
   group_by_all() %>% ungroup(market_value) %>%
   summarise(market_value = sum(market_value, na.rm = TRUE), .groups = "drop") %>%
-  write_csv(file = file.path(meta_output_dir, "20_Raw_Inputs", paste0(project_prefix, "_meta.csv")))
+  write.csv(file = file.path(meta_output_dir, "20_Raw_Inputs", paste0(project_prefix, "_meta.csv")))
```

Comment:
I think you're much better off using `readr::write_csv()` here because we can expect to have non-ascii characters in the portfolios we get, and `base::write.csv()` can have very uncertain behavior with that while `readr::write_csv()` has a much more consistent and reliable output (especially regarding file encodings).